com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -204,6 +204,22 @@ public void testTf2()
     testTf2("tf2_test_tensor_list.py", "add", 2, 3, 2, 3);
     testTf2("tf2_test_tensor_list2.py", "add", 0, 2);
     testTf2("tf2_test_tensor_list3.py", "add", 0, 2);
+    testTf2(
+        "tf2_test_model_call.py",
+        "SequentialModel.__call__",
+        0,
+        2); // NOTE: Change to testTf2("tf2_test_model_call.py", "SequentialModel.__call__", 1, 4,
+    // 2) once
+    // https://github.com/wala/ML/issues/24 is fixed.
+    testTf2(
+        "tf2_test_model_call2.py",
+        "SequentialModel.call",
+        0,
+        2); // NOTE: Change to testTf2("tf2_test_model_call.py", "SequentialModel.call", 1, 4, 2)
```

Comment:
[The problem]9https://github.com/ponder-lab/ML/pull/53/files#r1361414583 is on this line.